src/core/lv2/Lv2SubPluginFeatures.cpp

```diff
@@ -67,54 +69,128 @@ Lv2SubPluginFeatures::Lv2SubPluginFeatures(Plugin::Type type) :
 
 
 
-void Lv2SubPluginFeatures::fillDescriptionWidget(QWidget *parent,
-													const Key *k) const
+void addHbox(QWidget* parent, const char* left, QString right)
 {
-	const LilvPlugin *plug = getPlugin(*k);
-
-	auto label = new QLabel(parent);
-	label->setText(QWidget::tr("Name: ") + pluginName(plug));
+	if (right.isEmpty()) { return; }
 
-	auto label2 = new QLabel(parent);
-	label2->setText(QWidget::tr("URI: ") +
-		lilv_node_as_uri(lilv_plugin_get_uri(plug)));
-
-	auto maker = new QWidget(parent);
-	auto l = new QHBoxLayout(maker);
+	auto container = new QWidget(parent);
+	auto l = new QHBoxLayout(container);
 	l->setContentsMargins(0, 0, 0, 0);
 	l->setSpacing(0);
 
-	auto maker_label = new QLabel(maker);
-	maker_label->setText(QWidget::tr("Maker: "));
-	maker_label->setAlignment(Qt::AlignTop);
+	auto leftLabel = new QLabel(container);
+	leftLabel->setText(QWidget::tr(left));
+	leftLabel->setAlignment(Qt::AlignTop);
 
-	auto maker_content = new QLabel(maker);
-	maker_content->setText(
-		qStringFromPluginNode(plug, lilv_plugin_get_author_name));
-	maker_content->setWordWrap(true);
+	auto rightLabel = new QLabel(container);
+	if (right.startsWith("http") && !right.contains(' ') && !right.contains('\n'))
```

Comment:
I'm sure there are better ways to do URL detection, but practically speaking, this should work

Reply:
I tried to use [QUrl::isValid()](https://doc.qt.io/qt-6/qurl.html#isValid), but it recognizes "Calf Studio Gear" as a valid URL :smiley: For now, I will probably just keep it like that.